jupyter_packaging/setupbase.py

```diff
@@ -401,6 +402,29 @@ def run(self):
     return TargetsCheck
 
 
+def skip_if_exists(paths, CommandClass):
+    """Skip a command if list of paths exists."""
+    def should_skip():
+        return any(not Path(path).exist() for path in paths)
+    class SkipIfExistCommand(Command):
+        def initialize_options(self):
+            if not should_skip:
```

Comment:
Shouldn't this be?

```bash
if not should_skip():
```

Reply:
Good catch, and `all(Path(path).exist() for path in paths)` should improve readability

Reply:
Also wondering about the logic here.

Shouldn't the command be run (= not skipped) in case any of the files is missing?

Wondering whether this could be:

```python
    def should_run():
        return any(not Path(path).exist() for path in paths)
```

And then:

```python
if should_run:
    self.command = CommandClass(self.distribution)
    self.command.initialize_options()
else:
    self.command = None
```

Reply:
OK I can submit a follow-up PR then.